<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edge Module</string>
  </property>
  <widget class="QWidget" name="settings_widget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>980</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>980</height>
    </size>
   </property>
   <layout class="QGridLayout" name="grid_layout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontal_layout_video">
      <item>
       <layout class="QVBoxLayout" name="vertical_layout_player">
        <item>
         <layout class="QVBoxLayout" name="vertical_layout_video_viewer"/>
        </item>
        <item>
         <widget class="QGroupBox" name="group_box_player_controller">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="title">
           <string>Controller</string>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>640</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="button_start">
              <property name="maximumSize">
               <size>
                <width>210</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_settings">
              <property name="text">
               <string>Settings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_server_url_test">
              <property name="text">
               <string>Server URL test</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_ready">
              <property name="text">
               <string>Ready</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="edit_text_log">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>400</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="vertical_layout_monitoring"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
